nibabel/tests/test_diff.py

```diff
@@ -32,17 +33,58 @@
 from .test_parrec_data import BALLS, AFF_OFF
 from .test_helpers import assert_data_similar
 
+from hypothesis import given
```

Comment:
so hypothesis will need to be listed in dev-requirements.txt

Reply:
Done!